OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -135,6 +144,12 @@ int main()
         Model modelWithContacts("visualize_contacts.osim");
         populate_contactModelPrimitives(standard);
         testVisModelAgainstStandard(modelWithContacts, standard);
+        
+        // Model with WrapObjects
+        Model modelWithWrap("test_wrapAllVis.osim");
+        modelWithWrap.updDisplayHints().set_show_frames(false);
```

Comment:
I believe there's a bug where the flag is not universally applied/respected, I have a separate branch to investigate/fix but I don't want it entangled with this one.